mlflow/__init__.py

```diff
@@ -254,6 +254,7 @@
         ActiveModel,
         ActiveRun,
         autolog,
+        clear_active_model,
```

Comment:
can you add this here https://github.com/mlflow/mlflow/blob/cfcc87799e86f2836dce8cd2149518efa883d3a7/docs/api_reference/source/python_api/mlflow.rst#L60?